configs/config/debugging/pretrain/supervised/supervised_deit_b_integration_debug.yaml

```diff
@@ -20,7 +20,7 @@ config:
         - name: RandomResizedCrop
           size: 224
         - name: RandomHorizontalFlip
-        - name: VisslRandAugment
+        - name: RandAugment
```

Comment:
we should rebase to master. This change should be essentially reverted after the rebase.